lib/hawkular/prometheus/prometheus_api.rb

```diff
@@ -0,0 +1,50 @@
+require 'hawkular/base_client'
+require 'hawkular/inventory/entities'
+require 'ostruct'
+
+module Hawkular::Prometheus
+  class AlerterClient < Hawkular::BaseClient
```

Comment:
The `Client` part in this class name seems unnecessary, given that `Alerter` is already the role of the class.

Reply:
To be honest, I don't mind but I would mantain to indicate that this is a class which is client for an endpoint.